documentation/api/query/v4/aggregate-event-counts.markdown

```diff
@@ -26,45 +26,40 @@ Once this information is stored in PuppetDB, it can be queried in various ways.
 results into a single map.
 
 ### URL Parameters
 
-This endpoint builds on top of the [`event-counts`][event-counts] endpoint, and it uses all of the same URL parameters. The supported parameters are re-listed below for reference.
+This endpoint builds on top of the [`event-counts`][event-counts] endpoint, and it uses all of the same URL parameters. The supported parameters are listed below for easy reference.
 
-* `summarize_by`: Required. A string specifying which object types you'd like counted. Supported values are
-`resource`, `containing_class`, and `certname`, or any comma-separated
-combination thereof.
+* `summarize_by`: required. A string specifying which object types you'd like counted. Supported values are `resource`, `containing_class`, and `certname`, or any comma-separated combination thereof.
 
-* `query`: Optional. A JSON array of query predicates in prefix form (`["<OPERATOR>", "<FIELD>", "<VALUE>"]`).
-This query is forwarded to the [`events`][events] endpoint - see there for additional documentation.
-For general info about queries, see [the page on query structure.][query]
+* `query`: optional. A JSON array of query predicates in prefix form (`["<OPERATOR>", "<FIELD>", "<VALUE>"]`). This query is forwarded to the [`events`][events] endpoint - see there for additional documentation. 
+For general info about queries, see [our guide to query structure.][query]
 
-* `count_by`: Optional. A string specifying what type of object is counted when building up the counts of
-`successes`, `failures`, `noops`, and `skips`. Supported values are `resource` (default) and `certname`.
+* `count_by`: optional. A string specifying what type of object is counted when building up the counts of `successes`, `failures`, `noops`, and `skips`. Supported values are `resource` (default) and `certname`.
 
-* `counts_filter`: Optional. A JSON array of query predicates in the usual prefix form. This query is applied to
-the final event-counts output, but before the results are aggregated. Supported operators are `=`, `>`, `<`,
-`>=`, and `<=`. Supported fields are `failures`, `successes`, `noops`, and `skips`.
+* `counts_filter`: optional. A JSON array of query predicates in the usual prefix form. This query is applied to the final event-counts output, but before the results are aggregated. Supported operators are `=`, `>`, `<`, `>=`, and `<=`. Supported fields are `failures`, `successes`, `noops`, and `skips`.
 
-* `distinct_resources`: Optional.  (EXPERIMENTAL: it is possible that the behavior
-of this parameter may change in future releases.)  This parameter is passed along
-to the [`event`][events] query - see there for additional documentation.
+* `distinct_resources`: optional. (**Experimental: it is possible that the behavior
+of this parameter may change in future releases.**) This parameter is passed along
+to the [`events`][events] query, additional documentation about which can be found [here][events].
```

Comment:
this has two links to the same place. Wonder if we can chop it at the comma, or do something like "...is passed along to an events query. See the [events documentation][events] for more detail."